Scheduler/FirstComeFirstServe.c

```diff
@@ -0,0 +1,103 @@
+/**
+ * @file FirstComeFirstServe.c
+ */
+
+#include "FirstComeFirstServe.h"
+#include "ScheduleFile.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdbool.h>
+
+void RunFCFSProcess (ScheduleData* inputData){
+	bool debug = false;		
```

Comment:
are we keeping debug here?

Reply:
In the interest of time, we can leave it since it's set to false. For the next project, we should stick to using compile time conditional symbols (such as NDEBUG, _DEBUG, and the PRINT_PROCESSES_DATA macro I added) to control this kind of thing.